autogpt_platform/backend/pyproject.toml

```diff
@@ -69,6 +70,8 @@ cli = "backend.cli:main"
 format = "linter:format"
 lint = "linter:lint"
 test = "run_tests:test"
+migrate = "migrate.run_migrations:main"
+migrate_db = "migrate.run_migrations:main"
```

Comment:
what's the difference ?

Reply:
haha nothing, left it in accidentaly. can be removed